particles/Restaurants/source/ReservationForm.js

```diff
@@ -122,13 +121,22 @@ defineParticle(({DomParticle, html}) => {
         timePicker[`selected${i}`] = Boolean(partySize == i);
       }
       return {
-        subId: restaurantId,
+        subid: restaurantId,
         timePicker: {
           $template: 'time-picker',
           models: [timePicker]
-        }
+        },
+        partySize,
+        date
       };
     }
+    renderModel(model) {
+      super.renderModel({
+        modality: 'notification',
```

Comment:
in https://github.com/PolymerLabs/arcs/pull/3636 i propose to add `modality` to the rendering content in the slot-composer, so this won't be needed.

Reply:
We can discuss it. I'm not ready to jump there, because it reduces flexibility. As it is now, I can use one slot to transmit multiple modalities at will. To formalize it requires more steps.